src/flamegraph/merge.rs

```diff
@@ -74,7 +74,7 @@ fn flow<'a, LI, TI>(
     }
 }
 
-pub(super) fn frames<'a>(input: &'a str) -> (Vec<TimedFrame<'a>>, usize, usize) {
+pub(super) fn frames(input: &str) -> (Vec<TimedFrame>, usize, usize) {
```

Comment:
This change seems unrelated to this PR?

Reply:
Yes please, just to make it easier to review :)

Reply:
[Done](https://github.com/jonhoo/inferno/pull/32/commits/2ee9ca77c198766e29ee067561699564bcce49a0)